### ISSUE TITLE
publish 1.4.0

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#166]
The PhET-iO client would like a final production version (1.4.0) of ph-scale-basics by "late August".  RC testing will include both ph-scale and ph-scale-basics, for `brands=phet,phet-io`.

@arouinfar let's review open and blocking issues, and put together a timeline.

**Comment by pixelzoom** [comment]
At 7/2/2020 dev meeting, we noted that there are several sims (ph-scale, EFAC, SOM, ... others?) that all have the same "late August" deadline.  So this should be part of a coordinated effort.  See https://github.com/phetsims/tasks/issues/1040.

**Comment by arouinfar** [comment]
Looking at the open issues in this repo, it doesn't seem like there is anything sim-specific to be done before 1.4.0. I think we're also down to [just one blocking issue](https://github.com/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%3Ablocks-publication+label%3Adev%3Aphet-io+user%3Aphetsims+sort%3Acreated-asc+) in common code, and it's actively being worked on.

@pixelzoom if I'm missing something, or if you would like to chat, please let me know.

**Comment by pixelzoom** [comment]
In 7/9/2020 phet-io meeting and phetsims/tasks#1040, it was decided to send pH Scale through the RC process first, to shake out any PhET-iO issues.  Then we'll presumably use those shas for other sims that are part of the "late August" milestone. 

There's currently 1 blocking issue, https://github.com/phetsims/joist/issues/600.

**Comment by pixelzoom** [comment]
I ran into RC build problems on the build server. These issues are now blocking:

- [ ] https://github.com/phetsims/chipper/issues/940
- [ ] https://github.com/phetsims/chipper/issues/961

**Comment by pixelzoom** [comment]
I deleted ph-scale 1.4 branch, and will start over.

**Comment by pixelzoom** [comment]
I ran into RC build problems on the build server. I deleted ph-scale 1.4 branch, and will start over.

These issues are now blocking:

- [x] https://github.com/phetsims/chipper/issues/940
- [x] https://github.com/phetsims/chipper/issues/961

**Comment by pixelzoom** [comment]
RCs published:

ph-scale
phet: https://phet-dev.colorado.edu/html/ph-scale/1.4.0-rc.1/phet/ph-scale_all_phet.html
phet-io: https://phet-dev.colorado.edu/html/ph-scale/1.4.0-rc.1/phet-io/

ph-scale-basics
phet: https://phet-dev.colorado.edu/html/ph-scale-basics/1.4.0-rc.1/phet/ph-scale-basics_all_phet.html
phet-io: https://phet-dev.colorado.edu/html/ph-scale-basics/1.4.0-rc.1/phet-io

**Comment by pixelzoom** [comment]
RCs published 8/17/2020:

**pH Scale 1.4.0-rc.2**
QA issue: https://github.com/phetsims/QA/issues/533
phet: https://phet-dev.colorado.edu/html/ph-scale/1.4.0-rc.2/phet/ph-scale_all_phet.html
phet-io: https://phet-dev.colorado.edu/html/ph-scale/1.4.0-rc.2/phet-io/

**pH Scale: Basics 1.4.0-rc.2**
QA issue: https://github.com/phetsims/QA/issues/534
phet: https://phet-dev.colorado.edu/html/ph-scale-basics/1.4.0-rc.2/phet/ph-scale-basics_all_phet.html
phet-io: https://phet-dev.colorado.edu/html/ph-scale-basics/1.4.0-rc.2/phet-io

**Comment by pixelzoom** [comment]
In Slack phet-io channel 8/26/2020, there was a discussion how tracking/managing iO changes has become a problem, especially since the 1.4 release branches were created so long ago (7/9/2020) and the next RCs for ph-scale and ph-scale-basics now pushed out.   We decided to abandon the 1.4 branches, and start over with a new release branch (from master) when RC testing is resumed.

Closing this issue.

<details>
<summary>Slack discussion</summary>

Chris Malley  5:03 PM
For ph-scale, I suggest that we cut our losses,  plan to abandon ph-scale 1.4 and ph-scale-basics 1.4, and start over with new release branches.

Ariel Paul  5:03 PM
But that seems to be the likely correct approach for pH-Scale

Chris Malley  5:04 PM
I'll put it on the phet-io meeting agenda to confirm. If you really want me to totally ignore ph-scale until the next RC (which I'll be happy to do), it's the only way forward.

Ariel Paul  5:06 PM
I think at this point it is clearly the correct call, it will incur a QA cost, but it will probably be smart to test off master at that point anyhow (edited) 

@kathy agrees @pixelzoom so no need to be on the agenda. You have the confirmation

</details>
